docs/webserver/set-datetime.md

```diff
@@ -1,10 +1,11 @@
 # iRobot® Create® 3 Webserver - Set Date and Time
 The Set Date and Time page of the Create® 3 webserver allows the user to directy change the robot's date and time.
```

Comment:
Will this disable time sync even if it is connected to wifi?

Reply:
In the default configuration, effectively, yes, if the time jumps "enough" for the NTP daemon to assume that the times it's receiving from its sync sources are incorrect, which is why the NTP daemon has to be restarted after it is reconfigured.